tools/dhs/include/dhs_patch/dhs_patch_compat.h

```diff
@@ -131,4 +164,4 @@ dhs_a9_compat_s dhs_a9_compat_49 = {
 
 #endif /*DHS_PATCH_COMPAT_DATA*/
 
-#endif /*__DHS_PATCH_COMPAT_H*/
+#endif /*DHS_PATCH_COMPAT_H*/
```

Comment:
file.h and dhs_patch_compat.h have typedefs that always cause a redefinition of the IFILE type. I copied it to dhs_patch_compat.h to fix.